C/cxxwrap_julia/build_tarballs.jl

```diff
@@ -43,10 +44,11 @@ for basename in ["jlcxx_containers", "except", "extended", "functions", "hello",
   fullname = "lib"*basename
   push!(products, LibraryProduct(fullname, Symbol(fullname)))
 end
+products = expand_cxx_versions(products)
```

Comment:
Oh, well, it looks like we need to update a warning message...  This should be:
```suggestion
products = expand_cxxstring_abis(products)
```

Reply:
I just checked this, `expand_cxxstring_abis` is only defined for `Platform`s or `Vector{Platform}`. I pushed a new change, but I could still be wrong

Reply:
I also suspect something might go wrong and the expansion should be either filtered, considering only `cxx11` since it was what was already being done in [libcxxwrap-julia](https://github.com/JuliaInterop/libcxxwrap-julia/releases?after=v0.6.4). Same might be applicable for `gcc` versions vs `libgfortran`.

Reply:
Yes, you're right, `expand_cxxstring_abis` is something you apply to the platforms.  The warning message is completely wrong, we need to fix it.  I didn't even notice also the argument was wrong.